### ISSUE TITLE
I get an exception in Unity 2018.3.7f1

### ISSUE THREAD

**Issue by Temka193** [SpaceMadness/lunar-unity-console#135]
`[Lunar Mobile Console] Unable to list types for assembly: Accessibility, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
Exception of type 'System.Reflection.ReflectionTypeLoadException' was thrown.
  at (wrapper managed-to-native) System.Reflection.Assembly.GetTypes(System.Reflection.Assembly,bool)
  at System.Reflection.Assembly.GetTypes () [0x00000] in <d7ac571ca2d04b2f981d0d886fa067cf>:0
  at LunarConsolePluginInternal.ReflectionUtils.FindTypes (System.Reflection.Assembly assembly, LunarConsolePluginInternal.ReflectionTypeFilter filter) [0x00008] in D:\Projects\MyProject\Assets\LunarConsole\Scripts\Utils\ReflectionUtils.cs:274
UnityEngine.Debug:LogError(Object)
LunarConsolePluginInternal.Log:e(Exception, String) (at Assets/LunarConsole/Scripts/Utils/Log.cs:61)
LunarConsolePluginInternal.Log:e(Exception, String, Object[]) (at Assets/LunarConsole/Scripts/Utils/Log.cs:54)
LunarConsolePluginInternal.ReflectionUtils:FindTypes(Assembly, ReflectionTypeFilter) (at Assets/LunarConsole/Scripts/Utils/ReflectionUtils.cs:285)
LunarConsolePluginInternal.ReflectionUtils:FindAttributeTypes(Assembly, Type) (at Assets/LunarConsole/Scripts/Utils/ReflectionUtils.cs:261)
LunarConsolePluginInternal.ReflectionUtils:FindAttributeTypes(Assembly) (at Assets/LunarConsole/Scripts/Utils/ReflectionUtils.cs:256)
LunarConsolePlugin.LunarConsole:ResolveVariables() (at Assets/LunarConsole/Scripts/LunarConsole.cs:341)
LunarConsolePlugin.LunarConsole:InitPlatform(Int32, Int32, LunarConsoleSettings) (at Assets/LunarConsole/Scripts/LunarConsole.cs:245)
LunarConsolePlugin.LunarConsole:EnablePlatform() (at Assets/LunarConsole/Scripts/LunarConsole.cs:197)
LunarConsolePlugin.LunarConsole:OnEnable() (at Assets/LunarConsole/Scripts/LunarConsole.cs:140)`

**Comment by BoatWar** [comment]
Similarly on Unity 2018.3.9f1, and Lunar 1.6.1

```c#
[Lunar Mobile Console] Unable to list types for assembly: Accessibility, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
Exception of type 'System.Reflection.ReflectionTypeLoadException' was thrown.
  at (wrapper managed-to-native) System.Reflection.Assembly.GetTypes(System.Reflection.Assembly,bool)
  at System.Reflection.Assembly.GetTypes () [0x00000] in <d7ac571ca2d04b2f981d0d886fa067cf>:0 
  at LunarConsolePluginInternal.ReflectionUtils.FindTypes (System.Reflection.Assembly assembly, LunarConsolePluginInternal.ReflectionTypeFilter filter) [0x00008] in E:\Projects\Work\wwh\Assets\LunarConsole\Scripts\Utils\ReflectionUtils.cs:274 
UnityEngine.Debug:LogError(Object)
LunarConsolePluginInternal.Log:e(Exception, String) (at Assets/LunarConsole/Scripts/Utils/Log.cs:61)
LunarConsolePluginInternal.Log:e(Exception, String, Object[]) (at Assets/LunarConsole/Scripts/Utils/Log.cs:54)
LunarConsolePluginInternal.ReflectionUtils:FindTypes(Assembly, ReflectionTypeFilter) (at Assets/LunarConsole/Scripts/Utils/ReflectionUtils.cs:285)
LunarConsolePluginInternal.ReflectionUtils:FindAttributeTypes(Assembly, Type) (at Assets/LunarConsole/Scripts/Utils/ReflectionUtils.cs:261)
LunarConsolePluginInternal.ReflectionUtils:FindAttributeTypes(Assembly) (at Assets/LunarConsole/Scripts/Utils/ReflectionUtils.cs:256)
LunarConsolePlugin.LunarConsole:ResolveVariables() (at Assets/LunarConsole/Scripts/LunarConsole.cs:419)
LunarConsolePlugin.LunarConsole:InitPlatform(LunarConsoleSettings) (at Assets/LunarConsole/Scripts/LunarConsole.cs:323)
LunarConsolePlugin.LunarConsole:EnablePlatform() (at Assets/LunarConsole/Scripts/LunarConsole.cs:277)
LunarConsolePlugin.LunarConsole:OnEnable() (at Assets/LunarConsole/Scripts/LunarConsole.cs:220)
```

**Comment by weeeBox** [comment]
@AlexZonov, I'm preparing the fix. In a meantime, would you be able to run your project with `LUNAR_DEVELOPMENT` [custom define](https://docs.unity3d.com/Manual/PlatformDependentCompilation.html) and collect all log messages which look like this one: `[Lunar Mobile Console] Checking 'XXX, Version=X.X.X.X, Culture=XXX, PublicKeyToken=XXX'...`?

**Comment by weeeBox** [comment]
Fixed in 1.6.2
